code/api/test/user-email-test.js

```diff
@@ -0,0 +1,22 @@
+const chai = require('chai');
+
+const expect = chai.expect;
+const should = chai.should();
+const url = `http://localhost:8000/`;
+const request = require('supertest')(url);
+
+describe('GraphQL', () => {
+  it('Returns email address for user with id = 1', (done) => {
+    request.post('/graphql')
+    .send({ query: '{ user(id: 1) { email } }'})
+    .expect(200)
+    .end((err,res) => {
+        // res will contain array with one user
+        if (err) return done(err);
+        res.body.data.user.should.have.property('email')
+        // so this test hits our dev database (seeded from seeders file)
+        expect(res.body.data.user.email).to.eq('admin@crate.com')
+        done();
```

Comment:
It is very interesting to see chai/mocha used in the BE. That' being said, I understand what's going on! This looks great! I also appreciate the comments describing what will happen and what is expected.